gulpfile.js

```diff
@@ -88,7 +88,6 @@ if (argv.enable_sendmail) {
 var requireFilesExist = function(filePaths) {
   filePaths.forEach(function(filePath) {
     if (!fs.lstatSync(filePath).isFile()) {
-      console.log(filePath + ' is not a valid filepath, check spelling');
```

Comment:
I think these should be retained since they're in our setup scripts. Perhaps you could get them to throw an error or something instead, though. As long as the error is piped to the terminal, it's fine.

Reply:
Shall I change it to `console.error()`? Thanks!